méli-mélo/méli-mélo-en.html

```diff
@@ -3,105 +3,157 @@
 
-<div class="well">
-	<p>Features are grouped into <strong>compilations</strong> in order to quickly:</p>
+<div class="alert alert-info">
+	<p><strong>Tip to quickly get started!</strong> Code and/or expose your feature by leveraging <a href="https://github.com/wet-boew/gcweb-jekyll">GCWeb Jekyll theme</a>.</p>
+</div>
+
+<ol class="lst-spcd-2">
+	<li>Create a new feature folder inside the <a href="https://github.com/wet-boew/GCWeb/tree/master/m%C3%A9li-m%C3%A9lo"><code>/méli-mélo</code></a> GCWeb root folder.</li>
+	<li id="feature-name">Name your feature and its folder by following this notation: <code>YYYY-MM-[FeatureName]</code>. The year and month must correspond to the feature's initial publication date. For example "2021-05-steps".</li>
+	<li>Create and publish your demo / working example for each individual sub-features and styles, meaning each JS configurations and CSS classes respectively, either using the <a href="https://github.com/wet-boew/gcweb-jekyll">GCWeb Jekyll theme</a> or GCWeb itself.</li>
+	<li>Designate a <a href="#sponsor" rel="help">sponsor<sup aria-hidden="true"><span class="fas fa-info-circle"></span></sup></a> for the feature.</li>
+	<li>Build and publish an <a href="#implementation-plan" rel="help">implementation plan<sup aria-hidden="true"><span class="fas fa-info-circle"></span></sup></a>.</li>
+	<li>Ensure all <a href="#requirements">minimum requirements listed above</a> are met.</li>
+	<li><strong>Optional:</strong> Test your code by following the instructions on <a href="../docs/developing-en.html">developing for GCWeb</a>.</li>
+	<li>Submit your new feature through a GitHub Pull request (PR) in the GCWeb repository; please consult <a href="https://github.com/wet-boew/GCWeb/blob/master/CONTRIBUTING.md">contribution guidelines</a>.</li>
+	<li>If changes are requested on your PR after technical review (as per <a href="#tech-checklist">checklist below</a>), collaborate with the WET-BOEW team to get it approved.</li>
+	<li>Once your PR is approved, your feature will be assigned to a <a href="#mli-mlo-comp">méli-mélo compilation</a> and released on Canada.ca at the next release window between 1 to 3 weeks.</li>
+	<li><strong>Strongly recommended:</strong> After release, update the méli-mélo feature code by executing the implementation plan and addressing all TODO's identified by the WET-BOEW team.</li>
+	<li><strong>Recommended:</strong> Whenever possible, participate at the weekly <a href="https://github.com/wet-boew/wet-boew/wiki/WET-Office-hours,-Heures-de-service-de-la-BOEW">WET Office hours</a> on Tuesday afternoon.</li>
+</ol>
+
+<p class="mrgn-tp-lg">See <a href="2021-05-steps/index.html">2021-05-steps</a> and its <a href="https://github.com/wet-boew/GCWeb/tree/master/m%C3%A9li-m%C3%A9lo/2021-05-steps">folder on GitHub</a> as a complete example of a méli-mélo feature containing all the required information.</p>
+
+<details id="tech-checklist" class="mrgn-tp-lg mrgn-bttm-lg">
```

Comment:
Can I suggest to not hide that list, but we can probably put it in a well instead. That way it will kind of visually stand out from the other content in the page.